tests/automation_framework/tests/worker_tests/test_lookup.py

```diff
@@ -134,3 +134,31 @@ def test_worker_lookup_jsonrpc_different_version(self):
                 is TestStep.SUCCESS.value)
 
         logger.info('\t\t!!! Test completed !!!\n\n')
+
+
+    @pytest.mark.worker
+    @pytest.mark.worker_lookup
+    @pytest.mark.test_worker_lookup_diff_unit_length
+    @pytest.mark.listener
+    @pytest.mark.sdk
```

Comment:
This test case won't be applicable to SDK mode as lookup function takes only values from worker type enum.